Makefile

```diff
@@ -105,10 +105,14 @@ test:
 integration: controller-gen
 	@$(GO) test -v $(SOURCES_DIRS_GO) -coverprofile cover.out
 
-.PHONY: check
+.PHONY: clean
 clean: docker-image-clean
 	rm -f $(DEX_OPER_EXE) go.sum
 
+.PHONY: codecoverage
+codecoverage: 
```

Comment:
I think we can use `coverage`, it's shorter :grinning:

Reply:
yop thx changing

Reply:
@ereslibre sorry, yes, i forgot :rocket:

Reply:
fixed